aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -321,25 +381,32 @@ def create_publication(self, *, dbname: str) -> str:
             pub_options.append("TRUNCATE")
         pub_options = ",".join(pub_options)
         has_aiven_extras = self.has_aiven_extras(dbname=dbname)
-
+        pub_scope_logging = "all tables" if not only_tables else ",".join(only_tables)
         self.log.info(
-            "Creating publication %r for all tables in database %r, has_aiven_extras: %s", pubname, dbname, has_aiven_extras
+            "Creating publication %r for %s in database %r, has_aiven_extras: %s", pubname, pub_scope_logging, dbname,
+            has_aiven_extras
         )
         # publications as per database so connect to given database
         if has_aiven_extras:
-            self.c(
-                "SELECT 1 FROM aiven_extras.pg_create_publication_for_all_tables(%s, %s)",
-                args=(
-                    pubname,
-                    pub_options,
-                ),
-                dbname=dbname,
-                return_rows=0
-            )
+            if only_tables:
+                table_params = tuple(only_tables)
+                tables_subst = ",".join([" %s" for _ in only_tables])
```

Comment:
Nitpick: `[]` are redundant here, this will also work
```python
tables_subst = ",".join(" %s" for _ in only_tables)
```

Reply:
fixed